pipeline/scripts/sbom.ps1

```diff
@@ -1,16 +1,18 @@
 # Calculate the file name
-$sbomName = "SonarLint.visualstudio.sbom-$env:SONAR_PROJECT_VERSION.$env:BUILD_BUILDID-$(vsTargetVersion).json"
```

Comment:
Not all of the variable references had been converted to the `$env:xxx` format -> errors when building in `master`.